trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/common/MeansOfTransportHelper.java

```diff
@@ -1,28 +1,27 @@
-package uk.gov.defra.stw.mapping.toipaffs.chedpp;
+package uk.gov.defra.stw.mapping.toipaffs.common;
 
 import java.util.List;
 import java.util.Map;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.util.CollectionUtils;
 import uk.gov.defra.stw.mapping.dto.MainCarriageSpsTransportMovement;
-import uk.gov.defra.stw.mapping.toipaffs.common.MeansOfTransportMapper;
 import uk.gov.defra.tracesx.notificationschema.representation.MeansOfTransportAfterBip;
 import uk.gov.defra.tracesx.notificationschema.representation.enumeration.TransportMethod;
 
 @Component
-public class ChedppMeansOfTransportHelper {
-
-  private final MeansOfTransportMapper meansOfTransportMapper;
+public class MeansOfTransportHelper {
+  private final MeansOfTransportBaseMapper meansOfTransportBaseMapper;
 
   @Autowired
-  public ChedppMeansOfTransportHelper(MeansOfTransportMapper meansOfTransportMapper) {
-    this.meansOfTransportMapper = meansOfTransportMapper;
+  public MeansOfTransportHelper(
+      MeansOfTransportBaseMapper meansOfTransportBaseMapper) {
+    this.meansOfTransportBaseMapper = meansOfTransportBaseMapper;
   }
 
   public MeansOfTransportAfterBip map(
       List<MainCarriageSpsTransportMovement> data,
-      List<String> afterBcpSchemeIds,
+      List<String> beforeBcpSchemeIds,
```

Comment:
You've renamed this to "before" but the method still returns `MeansOfTransportAfterBip`?

Reply:
fixed